tests/copy.bats

```diff
@@ -139,3 +139,22 @@ load helpers
   buildah run $cid stat -c "%U:%G" /subdir
   test $(buildah run $cid stat -c "%U:%G" /subdir) = "nobody:root"
 }
+
+@test "copy-symlink" {
+  createrandom ${TESTDIR}/randomfile
+  ln -s ${TESTDIR}/randomfile ${TESTDIR}/link-randomfile
+
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json scratch)
+  root=$(buildah mount $cid)
+  buildah config --workingdir / $cid
+  buildah copy $cid ${TESTDIR}/link-randomfile
+  buildah unmount $cid
+  buildah commit --signature-policy ${TESTSDIR}/policy.json $cid containers-storage:new-image
+  buildah rm $cid
+
+  newcid=$(buildah from --signature-policy ${TESTSDIR}/policy.json new-image)
+  newroot=$(buildah mount $newcid)
+  test -s $newroot/randomfile
```

Comment:
Shouldn't we expect `link-randomfile` to keep its name when it's copied?

Reply:
I don't know, I will check with Docker.